emukit/examples/fabolas/continuous_fidelity_entropy_search.py

```diff
@@ -0,0 +1,80 @@
+from typing import Union
+
+import numpy as np
+
+from emukit.bayesian_optimization.acquisitions import ExpectedImprovement
```

Comment:
Some of these are relative, some absolute imports. I don't mind any inside examples folder, but you should probably pick one.

Reply:
ok switched all to absolute imports